src/cli/onefuzz/templates/regression.py

```diff
@@ -0,0 +1,250 @@
+            check_fuzzer_help,
+            fail_on_repro,
+        )
+
+    def libfuzzer(
+        self,
+        project: str,
+        name: str,
+        build: str,
+        pool_name: str,
+        *,
+        crashes: Container = None,
+        input_reports: Container = None,
+        inputs: Optional[Directory] = None,
+        target_exe: File = File("fuzz.exe"),
+        tags: Optional[Dict[str, str]] = None,
+        notification_config: Optional[NotificationConfig] = None,
+        target_env: Optional[Dict[str, str]] = None,
+        setup_dir: Optional[Directory] = None,
+        reboot_after_setup: bool = False,
+        target_options: Optional[List[str]] = None,
+        dryrun: bool = False,
+        duration: int = 24,
+        report_list: Optional[List[str]] = None,
+        crash_report_timeout: Optional[int] = None,
+        debug: Optional[List[TaskDebugFlag]] = None,
+        check_retry_count: Optional[int] = None,
+        check_fuzzer_help: bool = True,
+        fail_on_repro: bool = False,
+    ) -> Optional[Job]:
+
+        return self._create_job(
+            TaskType.libfuzzer_regression,
+            project,
+            name,
+            build,
+            pool_name,
+            crashes,
+            input_reports,
+            inputs,
+            target_exe,
+            tags,
+            notification_config,
+            target_env,
+            setup_dir,
+            reboot_after_setup,
+            target_options,
+            dryrun,
+            duration,
+            report_list,
+            crash_report_timeout,
+            debug,
+            check_retry_count,
+            check_fuzzer_help,
+            fail_on_repro,
+        )
+
+    def _create_job(
+        self,
+        task_type: TaskType,
+        project: str,
+        name: str,
+        build: str,
+        pool_name: str,
+        crashes: Container = None,
+        input_reports: Container = None,
+        inputs: Optional[Directory] = None,
+        target_exe: File = File("fuzz.exe"),
+        tags: Optional[Dict[str, str]] = None,
+        notification_config: Optional[NotificationConfig] = None,
+        target_env: Optional[Dict[str, str]] = None,
+        setup_dir: Optional[Directory] = None,
+        reboot_after_setup: bool = False,
+        target_options: Optional[List[str]] = None,
+        dryrun: bool = False,
+        duration: int = 24,
+        report_list: Optional[List[str]] = None,
+        crash_report_timeout: Optional[int] = None,
+        debug: Optional[List[TaskDebugFlag]] = None,
+        check_retry_count: Optional[int] = None,
+        check_fuzzer_help: bool = True,
+        fail_on_repro: bool = False,
+    ) -> Optional[Job]:
+
+        if not ((crashes and input_reports) or inputs):
+            self.logger.error(
+                "please specify either the 'crash' and 'input_reports' parameters or the inputs parameter"
+            )
+
+        if dryrun:
+            return None
```

Comment:
Do we really have to stop here, or can we do more while passing around the `dryrun` param?

Reply:
The dry run functionally has all but been phased out of the existing templates and is not part  of the managed templates.

At this point, it's primarily used for command validation.